src/config/credentials.ts

```diff
@@ -0,0 +1,12 @@
+export const mongoConfig = {
+  mongoUrl:
+    process.env.MONGO_URL ||
+    'mongodb://root:root@mongo/spotter?authSource=admin',
+}
+
+const getCustomRedisPort = () => process.env.REDIS_PORT || ''
```

Comment:
Not required

Reply:
This is required because otherwise it is giving number format exception with the string.